protocols.csv

```diff
@@ -23,9 +23,10 @@ code,	size,	name,	comment
 448,	0,	tls,	Transport Layer Security
 449,	V,	sni,	Server Name Indication RFC 6066 § 3
 454,	0,	noise,
-460,	0,	quic,
+460,  0,	quic,
```

Comment:
Did the formatting break?

Reply:
Yes. Editors break it all the time, since we're using tabs, and many editors convert those to spaces (including the built-in GitHub editor).